src/internal/rest.rs

```diff
@@ -185,9 +184,9 @@ impl<'a> Authorization<'a> {
 pub struct SignatureUrlString(String);
 
 impl SignatureUrlString {
-    pub fn new(encoded_full_url: &str) -> Result<SignatureUrlString, reqwest::UrlError> {
+    pub fn new(encoded_full_url: &str) -> Result<SignatureUrlString, url::ParseError> {
```

Comment:
did reqwest stop re-exporting this type or did you think the `url` one was better?

Reply:
They stopped.